Geometry/CSCGeometryBuilder/src/CSCGeometryParsFromDD.cc

```diff
@@ -200,6 +214,8 @@ bool CSCGeometryParsFromDD::build(const DDCompactView* cview,
     LogTrace(myName) << myName << ": fill fpar...";
     LogTrace(myName) << myName << ": dpars are... " << dpar[4] / cm << ", " << dpar[8] / cm << ", " << dpar[3] / cm
                      << ", " << dpar[0] / cm;
+    edm::LogVerbatim("CSCGeometryParsFromDD") << "(7) dpar[4]: " << dpar[4] / cm << " dpar[8]:  " << dpar[8] / cm
+                                              << " dpar[3]: " << dpar[3] / cm << " dpar[0]: " << dpar[0] / cm;
 
     fpar.emplace_back((dpar[4] / cm));
```

Comment:
Please delete line 24:
```
#include "CLHEP/Units/GlobalSystemOfUnits.h"
```
(We are trying to remove this file from all of Geometry code.)
Then these lines need to be changed like:
```
fpar.emplace_back(convertMmToCm(dpar[4]));
```

Reply:
@cvuosalo : done